### PR TITLE
Cat3 validation testing working with both good and bad XML files

### PR DESCRIPTION
**Pull request by emc4131**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

There was a failure caused by a schematron issue that had been addressed, but added tests were required to ensure this does not occur again. This PR addresses Cat3 schematron validation testing.

**Submitter:**
- [X] This pull request describes why these changes were made.
- [X] Internal ticket for this PR: [CYPRESS-368](https://jira.mitre.org/browse/CYPRESS-368)
- [X] Internal ticket links to this PR: [ONC Tracking 1540](https://oncprojectracking.healthit.gov/support/browse/CYPRESS-1540)
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: test/unit/qrda/cat3_test.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require_relative '../../test_helper'
+require 'cqm/models'
+require 'cqm_validators'
+
+module QRDA
+  module Cat3
+    class Cat3CVUTest < MiniTest::Test
```
Comment (dczulada) [inline test/unit/qrda/cat3_test.rb]: I would change the name of the class to be more generic.  'CVU' is a little to context specific for this library.

Name it something like Cat3ValidationTest

Comment (emc4131) [reply test/unit/qrda/cat3_test.rb]: I've changed the class name to Cat3ValidationTest.
